packages/tex-to-myst/src/tables.ts

```diff
@@ -127,7 +127,7 @@ export const TABLE_HANDLERS: Record<string, Handler> = {
       currentNode.rowspan = rowspan;
     }
     state.renderChildren(node.args[node.args.length - 1]);
-    state.closeNode();
+    state.closeParagraph();
```

Comment:
This PR also sneaks in a small bug fix with correctly closing multirow macro. It is closed differently than multicolumn macro below.